src/app_service/service/community_tokens/service.nim

```diff
@@ -47,32 +62,66 @@ QtObject:
   proc init*(self: Service) =
     self.events.on(PendingTransactionTypeDto.CollectibleDeployment.event) do(e: Args):
       var receivedData = TransactionMinedArgs(e)
-      # TODO signalize module about about contract state
-      if receivedData.success:
-        echo "!!! Collectible deployed"
-      else:
-        echo "!!! Collectible not deployed"
+      let deployState = if receivedData.success: DeployState.Deployed else: DeployState.Failed
+      let tokenDto = toCommunityTokenDto(parseJson(receivedData.data))
+      if not receivedData.success:
+        error "Collectible contract not deployed", address=tokenDto.address
+      try:
+        discard updateCommunityTokenState(tokenDto.address, deployState) #update db state
```

Comment:
Can this be done automatically in status-go instead? When the TX is done, we update the DB then send the signal?

Reply:
I was thinking about this and probably I will add this later.